frontend.js

```diff
@@ -209,6 +209,20 @@ module.exports = {
           ],
           allowSameModule: true,
         },
+        {
+          module: 'src/models',
+          allowReferenceFrom: [
+            'src/components',
```

Comment:
uiなどは依存させたくないので、こうしたいです！
```suggestion
            'src/components/model',
            'src/components/page',
```